packages/shared/src/lib/contexts/governance/utils/getNumberOfActiveProposals.ts

```diff
@@ -1,9 +1,15 @@
 import { get } from 'svelte/store'
 import { registeredProposalsForSelectedAccount } from '../stores'
 import { isProposalActive } from './isProposalActive'
+import { getProposalWithStatus } from '@contexts/governance'
+import { getL1Network } from '@core/network'
 
 export function getNumberOfActiveProposals(): number {
     const proposals = get(registeredProposalsForSelectedAccount)
-    const activeProposals = Object.values(proposals ?? {}).filter((proposal) => isProposalActive(proposal?.status))
+    const { currentMilestone } = getL1Network()
+    const activeProposals = Object.values(proposals ?? {}).filter((proposal) => {
+        const { status } = getProposalWithStatus(proposal, get(currentMilestone))
+        return isProposalActive(status)
+    })
```

Comment:
To only access the currentMilestone store once:

```suggestion
    const currentMilestone = get(getL1Network().currentMilestone)
    const activeProposals = Object.values(proposals ?? {}).filter((proposal) => {
        const { status } = getProposalWithStatus(proposal,currentMilestone)
        return isProposalActive(status)
    })
```